distribution/sender/routes/debezium-route.xml

```diff
@@ -1,29 +1,37 @@
 <routes xmlns="http://camel.apache.org/schema/spring">
-    <route id="debezium" errorHandlerRef="outBoundErrorHandler">
+    <route id="debezium" errorHandlerRef="noErrorHandler">
```

Comment:
Or a special errorHandler route that can properly record/log the event, because if this route ever fails, we have a problem because we lose the event.

Reply:
What kind of error we can have here? And what if the errorHandlerRef route fails too??